back/scripts/test-utils/select_test_citites.py

```diff
@@ -0,0 +1,62 @@
+"""This script is used to generate a test sample for the OFGL cities.
```

Comment:
Because this file's entire purpose is to update test fixture, I would put it in ./tests/back/....